app/src/main/java/org/metabrainz/android/presentation/features/search/SearchActivity.kt

```diff
@@ -164,14 +170,24 @@ class SearchActivity : AppCompatActivity(), SearchView.OnQueryTextListener {
     }
 
     override fun onQueryTextSubmit(query: String): Boolean {
+        if (!App.context!!.isOnline){
```

Comment:
We need to stop force-unwrapping things. 
Try something like `App.context?.isOnline == false` over here.

Reply:
Changed the if condition to ```App.context?.isOnline == false```.